lib/Rozier/src/Controllers/Nodes/NodesController.php

```diff
@@ -191,7 +191,7 @@ public function editAction(Request $request, int $nodeId, ?int $translationId =
             'nodeName' => $node->getNodeName(),
         ]);
         try {
-            if ($node->getNodeType()->isReachable() && !$node->isHome()) {
+            if ($this->nodeTypesBag->get($node->getNodeTypeName())->isReachable() && !$node->isHome()) {
```

Comment:
```suggestion
            if ($this->nodeTypesBag->get($node->getNodeTypeName())?->isReachable() && !$node->isHome()) {
```

Do not forget nullish operator when you chain `NodeTypes::get` method